homework-6/script-1.js

```diff
@@ -0,0 +1,83 @@
+//Написати функцію, яка приймає 4 аргументи і шукає найменше число серед них
+const getMinNumber = (num_1, num_2, num_3, num_4) => {
+    if (!isNaN(num_1) && !isNaN(num_2) && !isNaN(num_3) && !isNaN(num_4) && num_1 !== null && num_2 !== null && num_3 !== null && num_4 !== null && num_1 !== undefined && num_2 !== undefined && num_3 !== undefined && num_4 !== undefined) {
+        const result = Math.min(num_1, num_2, num_3, num_4);
+        return result;
+    } else {
+        console.log("Unfortunately, I can't figure out min number");
+    }
+}
+getMinNumber("5", 0, 6, 2);
+
+/*Написати функцію, яка приймає 2 аргументи та шукає найбільший спільний дільник для них. 
+Якщо такого числа немає, то повідомляти про відсутність*/
+const getGreatestCommonDivisor = (num_1, num_2) => {
```

Comment:
Seems to me it should also validate zero

<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191562773-108262af-3974-462a-a4b4-1dca20eac83a.png">